deployment/src/main/java/com/datastax/oss/quarkus/deployment/internal/CassandraClientProcessor.java

```diff
@@ -276,9 +276,8 @@ CassandraClientBuildItem cassandraClient(
   @BuildStep
   HealthBuildItem addHealthCheck(CassandraClientBuildTimeConfig buildTimeConfig) {
     return new HealthBuildItem(
-        "com.datastax.oss.quarkus.runtime.internal.health.CassandraHealthCheck",
-        buildTimeConfig.healthEnabled,
-        "cassandra");
+        "com.datastax.oss.quarkus.runtime.internal.health.CassandraAsyncHealthCheck",
```

Comment:
It seems that it is safe to switch to Async implementation. It has a dependency on `Uni`, but this type is always present in the `cassandra-quarkus-deployment` module:
```
    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-mutiny-deployment</artifactId>
    </dependency>
```

Reply:
This _seems_ right.  How would this come into play with transitive exclusions thought?  I'm thinking of something like a customer who doesn't want to write reactive code in their app and thus excludes Mutiny as a transitive dependency... and then that customer unexpectedly loses access to health checks.

Reply:
Hmmm, that's probably a non-issue.  Forgot QuarkusCqlSession already extends a number of interfaces which expose Uni/Multi directly... so I suspect you'd run aground well before you go to this point.

Reply:
yes, I was trying to make it NOT work by excluding mutiny, and reactive dependencies in the `integration-tests` and `quickstart` and it was working properly. So it seems that it is safe to make `CassandraAsyncHealthCheck` default